test/test_centroids.py

```diff
@@ -0,0 +1,21 @@
+import os
+from unittest import TestCase
+import numpy as np
+import xarray as xr
+import uxarray as ux
+from pathlib import Path
+from uxarray.grid.coordinates import _centroid_from_mean_verts
+
+current_path = Path(os.path.dirname(os.path.realpath(__file__)))
+
+
+class TestCentroids(TestCase):
+
+    def test_centroids_from_mean_verts(self):
+        path = xr.open_dataset(current_path / "meshfiles" / "mpas" / "QU" /
+                               "mesh.QU.1920km.151026.nc")
+
+        # Open the dataset and find the centroids
+        grid = ux.open_grid(path)
+        _centroid_from_mean_verts(grid)
+        self.assertIsNotNone(grid.Mesh2_face_x)
```

Comment:
Sure, what would that look like? Since I don't have values to compare, should I just take the mean and compare it? Seems slightly redundant but I am not sure how else to check for validity.

Reply:
Yeah, I would make a simple face, one with four known vertices, and then run the algorithm to see if it matches the expect centroid from your manual computations.

Reply:
Something like this, from the remapping PR

https://github.com/UXARRAY/uxarray/blob/e3f9e0482e99a0180b09d1087716f3d633009cb2/test/test_remap.py#L26-L57